source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md

```diff
@@ -1,53 +1,100 @@
+
+
+{% anchor h3 %} Text that looks like a number {% endanchor h3 %}
+
+Make sure that you don’t have text fields that look like numbers. Text that look like numbers are those that contain punctuation characters like a dash, underscore, parenthesis, or multiple dots.
+
+```
 - US Zip Code: 80202-1713
 - Phone numbers: “(555) 555-5555” or “555.555.5555”
 - Monetary Values with the currency indicator such as $3.50 or €5.73
+```
+{% anchor h3 %} CSV Headers can be blank {% anchor h3 %}
 
-**Data formatting**
-
-Make sure that your dates are properly formatted in your CSV. If you created your CSV with Excel, you may need to force the date format before exporting to CSV. The date format needs to be one of the following: MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY
+- Header fields can be blank in your CSV, but must be named and have their type defined later during the upload process.
 
-**Missing Header**
+![]({{root_url}}/images/listupload_5.png)
 
-Your CSV should always have a header row. The rules for this are:
+{% anchor h3 %} Date formatting {% endanchor h3 %}
 
-1. You must have an “email” column.
-2. Capitalization doesn’t matter.
-3. The other columns can be blank.
+Make sure that your dates are properly formatted in your CSV. If you created your CSV with Excel, you may need to force the date format before exporting to CSV. The date format needs to be one of the following: `MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY`
 
-**Custom fields in your CSV but not your account**
+{% anchor h3 %} Custom fields in your CSV but not your account {% endanchor h3 %}
 
 If you include a column in your CSV that is for a custom field not defined in your account, then the CSV will be rejected. We are currently working to add a feature that will let you set up custom fields as a step during the upload process.
 
-**I have more than one of the same email in my database**
+{% anchor h3 %} I have more than one of the same email in my database {% anchor h3 %}
 
 With SendGrid’s Marketing Campaigns feature, the unique identifier is the email address. So, if you upload the same email address multiple times, the custom field data associated with that email will be updated with each upload to the most recently uploaded/updated information. This feature helps prevent you from accidentally emailing the user after they have unsubscribed.
 
 What you may want to do is add logic to set a custom field, based on the reason why you have duplicate emails in your system (for example multiple product lines) and then segment your user to be in lists based on those custom fields.
 
 **‹ ** [Custom Fields](https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/custom_fields.html)
 
+
+{% anchor h2 %} Specific Upload Errors {% endanchor h2 %}
+
```

Comment:
Can you add a blurb here explaining these errors? Something like: Here is a list of potential errors you many encounter along with some easy fixes to help you successfully upload a CSV file. (this is terrible, I'm sure you can come up with something better)